### ISSUE TITLE
ClaymoreDual doesn't support SIA option

### ISSUE THREAD

**Issue by ShrinivasKR** [nicehash/NiceHashMiner-Archived#986]
There isn't an option in the NiceHash miner application to dual mine Siacoin, although the version of the miner that it is using does in fact support dual mining Ethereum and Sia using Nicehash Stratum.

**Comment by billmcilhargey** [comment]
Once SIA is implemented into this NiceHash Miner then @DillonN mentioned this should be an easy implementation

Reference: https://github.com/nicehash/NiceHashMiner/pull/919

**Comment by DillonN** [comment]
Yes when there is official SIA support in any capacity I can submit a pull request easily for this

**Comment by ShrinivasKR** [comment]
Good to hear.
Right now, I am running it from the command line, connecting to nicehash ethereum and nicehash sia pools.

**Comment by DillonN** [comment]
I've submitted a pull request for this #985

**Comment by ShrinivasKR** [comment]
Awesome!
